INSTALL.txt

```diff
@@ -56,18 +57,35 @@ Running WSO2 API Manager in standalone mode
                 -Dweb.location	The directory into which UI artifacts
                                 included in an Axis2 AAR file are extracted to
 
+
+Secure sensitive information in carbon configuration files
+----------------------------------------------------------
+
+There are sensitive information such as passwords in the carbon configuration.
+You can secure them by using secure vault. Please go through following steps to
+secure them with default mode.
+
+    On Linux: ./ciphertool.sh -Dconfigure
+    On Windows: ./ciphertool.bat -Dconfigure
+
+For more details see
+    https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/logins-and-passwords/working-with-encrypted-passwords/
```

Comment:
Isn't it ideal to link to the specific doc-version path itself? 
i.e. https://apim.docs.wso2.com/en/3.1.0/install-and-setup/setup/security/logins-and-passwords/working-with-encrypted-passwords/

Reply:
@bhathiya

Reply:
yes, we should do so whenever we use links in docs.

Reply:
Fixed https://github.com/wso2/product-apim/pull/8971/commits/061ead8cf43422243492a81c383c1681db180714

Reply:
@Sarangan0219 pls fix this inn release-notes.html too.

Reply:
Fixed https://github.com/wso2/product-apim/pull/8971/commits/0fdb4d83b18bce93b8bc175a80ba28c40261a25b